src/animations/scrollAnimation.js

```diff
@@ -0,0 +1,32 @@
+import gsap from 'gsap';
+import ScrollMagic from 'scrollmagic';
+import { ScrollMagicPluginGsap } from 'scrollmagic-plugin-gsap';
+
+ScrollMagicPluginGsap(ScrollMagic, gsap);
+
+const onScrollAnimate = (animatedSectionWrapper, id) => {
+  const elements = animatedSectionWrapper.current.children;
+
+  const appearTween = gsap.fromTo(
+    elements,
+    { y: '+=200', autoAlpha: 0 },
+    {
+      duration: 1,
+      ease: 'bounce.out',
```

Comment:
it's proper name for `easing` ? Can you provide link to docs?

Reply:
it works on dev server so it must be ok :) link: https://greensock.com/docs/v3/Eases